### PR TITLE
ES003

### PR DESCRIPTION
**Pull request by kbrizzle**
Implements and tests the proposal for the new parameter-updated GovernorAlpha [here](https://github.com/emptysetsquad/emptyset/pull/31).

### CODE REVIEW

File: proposals/emptyset/es003.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import { ethers } from 'ethers'
+import { EMPTYSET_CONTRACTS } from './contracts'
+import { Proposal } from '../../test/testutil/govern'
+
+const PROPOSAL_TEXT = `# Enabling Cheaper L1 / L2 Wrapping
```
Comment (arjun-io) [inline proposals/emptyset/es003.ts:5]: We should update this text 🙏

File: test/integration/EmptySetProposal003.test.ts [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+import { EMPTYSET_CONTRACTS } from '../../proposals/emptyset/contracts'
+import { ES_003 } from '../../proposals/emptyset/es003'
+
+const { ethers, deployments } = HRE
+const PROPOSER_ADDRESS = '0x589CDCf60aea6B961720214e80b713eB66B89A4d' // Equilibria Multisig
+
+const USE_REAL_DEPLOY = false
+const FORK_BLOCK = 14994972
+
+describe.only('Empty Set Proposal 003', () => {
```
Comment (arjun-io) [inline test/integration/EmptySetProposal003.test.ts:26]: nit: remove `only`

Comment (arjun-io) [reply test/integration/EmptySetProposal003.test.ts:26]: Also, can we pull the constant values that were changed to ensure they are set to what we expect?

File: proposals/emptyset/es003.ts (hunk 1 above)
Comment (kbrizzle) [reply proposals/emptyset/es003.ts:5]: updated 🔥, we might need to update this again per forum feedback as well.

File: test/integration/EmptySetProposal003.test.ts (hunk 2 above)
Comment (arjun-io) [reply test/integration/EmptySetProposal003.test.ts:26]: nit: still have an `only` here 😅

**Review by arjun-io — APPROVED** [review]

**Pull request merged**